.zenodo.json

```diff
@@ -130,6 +154,10 @@
       "affiliation":"TU Dresden",
       "type": "Other"
     },
+    {
+      "name": "Worpitz, Benjamin",
+      "affiliation": "LogMeIn Inc."
```

Comment:
The type others is missing and is required any name in contributors: `"type": "Other"`

Reply:
Done